explore/src/main/scala/explore/config/AdvancedConfigurationPanel.scala

```diff
@@ -365,79 +367,49 @@ sealed abstract class AdvancedConfigurationPanelBuilder[
     ).clearable(^.autoComplete.off)
 
   val component =
-    ScalaFnComponent
-      .withHooks[Props]
-      .useContext(AppContext.ctx)
-      // .useStateViewBy { (props, ctx) =>
-      //   import ctx.given
-
-      //   overrideExposureTimeMode(props.observingMode).get
-      //     .map(ExposureTimeModeType.fromExposureTimeMode)
-      // }
-      // .useEffectWithDepsBy { (props, ctx, _) =>
-      //   import ctx.given
-
-      //   overrideExposureTimeMode(props.observingMode).get.map(
-      //     ExposureTimeModeType.fromExposureTimeMode
-      //   )
-      // }((_, _, exposureModeEnum) =>
-      //   newExpModeEnum =>
-      //     if (exposureModeEnum.get =!= newExpModeEnum) exposureModeEnum.set(newExpModeEnum)
-      //     else Callback.empty
-      // )
-      // filter the spectroscopy matrix by the requirements that don't get overridden
-      // by the advanced config (wavelength, for example).
-      .useMemoBy((props, _) =>
-        (props.spectroscopyRequirements.get.focalPlane,
-         props.spectroscopyRequirements.get.capability,
-         props.spectroscopyRequirements.get.focalPlaneAngle,
-         props.spectroscopyRequirements.get.resolution,
-         props.spectroscopyRequirements.get.wavelengthCoverage,
-         props.confMatrix.matrix.length
-        )
-      ) { (props, _) =>
-        { case (fp, cap, fpa, res, rng, _) =>
-          props.confMatrix.filtered(
-            focalPlane = fp,
-            capability = cap,
-            slitLength = fpa.map(s => SlitLength(ModeSlitSize(s))),
-            resolution = res,
-            range = rng
-          )
-        }
-      }
-      // Try to find the mode row from the spectroscopy matrix
-      .useMemoBy { (props, ctx, rows) =>
-        import ctx.given
-
-        (props.spectroscopyRequirements.get.wavelength,
-         rows,
-         centralWavelength(props.observingMode).get,
-         grating(props.observingMode).get,
-         filter(props.observingMode).get,
-         fpu(props.observingMode).get
-        )
-      } { (props, _, _) =>
-        { case (reqsWavelength, rows, _, _, _, _) =>
-          findMatrixData(props.observingMode.get, reqsWavelength, rows)
-        }
-      }
-      .useStateView(ConfigEditState.View)
-      .render { (props, ctx, _, modeData, editState) =>
+    ScalaFnComponent[Props]: props =>
+      for
+        ctx       <- useContext(AppContext.ctx)
+        emv       <- useStateView(
```

Comment:
I'm a little confused. Does this mean the exposure mode drop down changes this view but doesn't really change anything?